app/actors/curation_concerns/actors/apply_order_actor.rb

```diff
@@ -3,11 +3,28 @@ module Actors
     class ApplyOrderActor < AbstractActor
       def update(attributes)
         ordered_member_ids = attributes.delete(:ordered_member_ids)
+        sync_members(ordered_member_ids)
         apply_order(ordered_member_ids) && next_actor.update(attributes)
       end
 
       private
 
+        def sync_members(ordered_member_ids)
+          return true if ordered_member_ids.nil?
+          existing_members_ids = curation_concern.members.select { |k| k.class.to_s != 'FileSet' }.map(&:id)
+          (existing_members_ids - ordered_member_ids).each do |old_id|
+            work = ::ActiveFedora::Base.find(old_id)
+            curation_concern.ordered_members.delete(work)
+            curation_concern.save
```

Comment:
Do you think we could extract it from the loop and just save once after it finishes iterating through the id's?